### PR TITLE
update data.json (#310)

### PR DESCRIPTION
**Pull request by An-prog-hub**
An-prog-hub

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -520,5 +520,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
-  }
+  },
+  {
+    "name": "Ankush Chauhan",
+    "githubUsername": "An-prog-hub",
+    "imageUrl": "https://user-images.githubusercontent.com/67872399/106575972-a71f9f80-6562-11eb-87dc-1b97d93f4d71.png",
+    "issueNumber": 310
+  },
```
Comment (AllanRegush) [inline data.json:529]: I don't think we need a comma here. :nerd_face:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment :smile:

**Comment by AllanRegush** [comment]
@An-prog-hub Just Checking in. Do you need any help? If so, let me know :nerd_face:

**Comment by AllanRegush** [comment]
@An-prog-hub Just Checking in again. Do you need any help? If so, let me know 🤓

File: data.json (hunk 1 above)
Comment (Vyvy-vi) [inline data.json:529]:
```suggestion
  }
```

Comment (Vyvy-vi) [reply data.json:529]: As Allan said, the comma could be removed 🤓 (I think that should fix the prettier fails)

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:

**Pull request merged**